Automattic/newspack-theme
# Replace hue-based colour picker with hex-based colour picker

### All Submissions:


### Changes proposed in this Pull Request:

Twenty Nineteen (the theme the Newspack theme is based on) has one custom colour; it uses a hue-specific picker in the Customizer, to allow users to pick a colour's hue, and controls its saturation and lightness, largely using the hsl value throughout the theme.

The purpose is to ensure that someone can't pick a colour with insufficient contrast, since it's used for links, the menu, and button.

For our purposes, a colour picker that allows picking a specific hex colour makes more sense -- it allows finer control over the exact shade selected, to make it easier to match existing branding. 

This update swaps the existing hue-based colour picker with one that allows you to pick colours by hex value. It also updates the associated code, including removing unused filters. 

At this time, for simplicity, I've also removed the darker colour variation that's generated for the editor. As a next step, this will need to be re-added, as well as the ability to pick secondary and tertiary colours. 

See #17.

### How to test the changes in this Pull Request:

1. Apply the patch.
2. In the Customizer, change the custom colour and make sure it previews correctly.
3. Save the changes, and make sure they're reflected on the front-end.
4. Edit a post or page and add a block with colour settings (like the paragraph block). Make sure that the custom colour shows up as one of the colour options. 

### Other information:
